python/CMakeLists.txt

```
@@ -54,6 +54,20 @@ target_compile_definitions(python_bindings PRIVATE
     HAVE_SNPRINTF
 )
 
+nanobind_add_stub(
+    bindings_stub
+    MODULE bindings
+    OUTPUT stubs/bindings.pyi
+    PYTHON_PATH $<TARGET_FILE_DIR:python_bindings>
+    DEPENDS bindings
+)
+
+add_custom_target(
+    dynamic_stubs ALL
+    COMMAND ${Python_EXECUTABLE} ${CMAKE_SOURCE_DIR}/python/novatel_edie_customizer/novatel_edie_customizer/stubgen.py ${CMAKE_SOURCE_DIR}/database/database.json ${CMAKE_CURRENT_BINARY_DIR}/stubs
+    COMMENT "Generating dynamic stubs"
```

```suggestion
    COMMENT "Generating dynamic stubs"
    DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}/novatel_edie_customizer/novatel_edie_customizer/stubgen.py
```
Re-generate the stubs automatically when `stubgen.py` is modified. Makes development more convenient.